projects/openvswitch/fuzzer.options

```diff
@@ -0,0 +1,3 @@
+[libfuzzer]
+close_fd_mask = 3
```

Comment:
Just realize that this will prevent assert messages from printing (maybe you don't need them?)

Reply:
This was a trade-off for filtering stdout/err spam for these targets.

Reply:
ACK